examples/direct_fidelity_estimation.ipynb

```diff
@@ -222,22 +222,9 @@
   }
  ],
  "metadata": {
```

Comment:
It does not look like there are meaningful changes to this file relative to the PR -- probably we should just remove it.